score.py

```diff
@@ -103,8 +103,14 @@ def main(
 
     scores = {}
     status = "INVALID"
-    with open(output_file, encoding="utf-8") as out:
-        res = json.load(out)
+    try:
+        with open(output_file, encoding="utf-8") as out:
+            res = json.load(out)
+    except (FileNotFoundError, json.decoder.JSONDecodeError):
+        res = {
+            "validation_status": "",
+            "validation_errors": "",
```

Comment:
Looks good! I'm seeing that this empty dictionary will trigger a call to the `else` condition in the `if/else` statement below and run:

```
errors = "Submission could not be evaluated due to validation errors."
```

which makes sense to me!

What do you think of also surfacing the exact error either on Synapse or in the Seqera execution logs, so we know when a failed scoring happened because of a failed validation, or a misformatted results JSON? Since this would be a mistake on the challenge organizers' end, I'd be fine with just adding a print statement in the `except` so it gets surfaced in the Seqera execution logs.

Reply:
Ohhh, good point!  I was thinking on this more, and what do you think about the script attempting to score anyway when `validation_status` is empty? If it fails during scoring then the error message would be:

```
"Error encountered during scoring; submission not evaluated."
```

Reply:
Or would you prefer an explicit "log" that indicates scoring failed and no validation was done beforehand?

(which _could_ be implied by empty `validation_*` values?)

Reply:
Yes, I like that idea! The error would then get surfaced as a submission annotation which the users/organizers can see. I would only suggest to still add a print statement in the `except` so it's easier for us to spot when it's a formatting issue

Reply:
Perfect, thank you! LGTM! This is ready to go 🔥